arrays.js

```diff
@@ -0,0 +1,50 @@
+const colors=[ "orange", "blue", "green", "black"];
+//  for (const color of colors){
+//      console.log(color);
+//  }
+
+// colors.forEach(item => console.log(item));
+
+// let number = 3;
+// function rotateRight(number, colors){
+//     for(i=0; i<number; i++){
+//         let last = colors.pop();
+//         colors.unshift(last);
+//     }
+// }
+// rotateRight(number, colors);
+// console.log(colors);
+
+// const numbers= [2,3,5,7,21,64];
+// const initalValue = 50;
+// const sumAll = numbers.reduce((accumulator, currentValue) => accumulator + currentValue,initalValue);
+// console.log(sumAll);
+
+// const number=10;
+// const array = [];
+// function countPiramide(number, array){   
+//     for (i=0; i<number; i++)
+//         {array.push(i)}
+//         const array2 = [...array];
+//         const reve = array2.reverse();
```

Comment:
Da li je sve ovo trebalo biti unutar for-a ili ?
```suggestion
//     for (i=0; i<number; i++){
//         array.push(i)
//         const array2 = [...array];
//         const reve = array2.reverse();
//         }
```

Ovako nesto? Ako nije, onda mozemo i ovako napisati: 

```suggestion
//     for (i=0; i<number; i++)
//         array.push(i)
//      const array2 = [...array];
//      const reve = array2.reverse();
```

bez {} for ce znati da gleda samo prvu liniju ispod